db/migrate/20191120015344_add_timezone_to_user_options.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AddTimezoneToUserOptions < ActiveRecord::Migration[6.0]
+  def up
+    add_column :user_options, :timezone, :string, index: true
```

Comment:
Curious why you'd like to index the timezone column here. Wouldn't we always want to retrieve it by `user_id`?

Reply:
Yeah I suppose you are right, the only thing it would be useful for is retrieving all the users in a timezone in a query, which I'm not sure we need right now. I can remove it and we can always put it back in later if we need